src/vma/sock/sockinfo.h

```diff
@@ -206,6 +206,7 @@ class sockinfo : public socket_fd_api, public pkt_rcvr_sink, public pkt_sndr_sou
 
 	lock_spin_recursive	m_lock_rcv;
 	lock_mutex		m_lock_snd;
+	lock_mutex_recursive	m_rx_migration_lock;
```

Comment:
Do you see any reason to use recursive lock? Please consider using usual lock.

Reply:
Updated